feature/home/src/main/kotlin/com/nexters/bandalart/android/feature/home/HomeScreen.kt

```diff
@@ -189,15 +189,17 @@ internal fun HomeScreen(
   }
 
   if (uiState.isBandalartListBottomSheetOpened) {
-    BandalartListBottomSheet(
-      bandalartList = uiState.bandalartList,
-      currentBandalartKey = uiState.bandalartDetailData?.key,
-      getBandalartDetail = getBandalartDetail,
-      setRecentBandalartKey = setRecentBandalartKey,
-      showSkeletonChanged = showSkeletonChanged,
-      onCancelClicked = { openBandalartListBottomSheet(false) },
-      createBandalart = createBandalart,
-    )
+    uiState.bandalartDetailData?.key?.let {
```

Comment:
```suggestion
uiState.bandalartDetailData?.let {
```
DetailData가 null 아니면 key 는 null 일수가 없을꺼라 {} 블럭 내에서 그냥 key!! 처리해줘도 될것같아 
api 구조상 반드시 key 는 들어오니깐